api/connectionmanager/client.go

```diff
@@ -42,7 +42,13 @@ func (c *ConnectionManager) GetConnections(opts ...filters.Option) (*response.Re
 	connections := &response.ResultSet[Connection]{}
 	params := url.Values{}
 
-	for _, opt := range opts {
+	// Set default options, which will be overwritten by opts if defined.
+	options := append([]filters.Option{
+		filters.Paging(0, 50),
```

Comment:
Default params by UI app is: 

connection-manager/api/v1/connections/search?offset=0&limit=25&sortkey=connected&sortdir=desc&fuzzycount=true

Thus, sortkey, sortDir and fuzzycount should be similar